client/network/src/config.rs

```diff
@@ -384,7 +384,7 @@ impl NetworkConfiguration {
 			"test-node",
 			"test-client",
 			Default::default(),
-			&std::env::current_dir().expect("current directory must exist"),
+			Some(std::env::current_dir().expect("current directory must exist")),
```

Comment:
Check the commit history but I think this can be `None`. Those are used only in some specific test I think

Reply:
Actually you don't even need to check the logs. As @tomaka said, it wasn't used at all anywhere so you can replace all of them to None instead of the current directory

Reply:
Oh yeah, that makes sense.